### ISSUE TITLE
Create more projects on BOINC

### ISSUE THREAD

**Issue by Bender250** [crocs-muni/EACirc#176]
Find out, how to create more EACirc_* applications on BOINC, and create them to allow parallel versions computation.

Discuss with Mirek, Martin U. and also try Martin S. project.

Or debug what are the bugs in actual EACirc_dev and EACirc_SHA3.

//Get more privileges on Boinc administration.

**Comment by mathius** [comment]
I know exactly, how to do that (currently, I have sufficient privileges to do so, but you don't). We tried to use it some time ago, but the new application was as useless as the other two that are currently unused. Made a conclusion to wait for the new BOINC with regard to this matter.

**Comment by Bender250** [comment]
From reason, that Martin wrote, I just tested already created projects and tried to find an unused and working one. The result of the research is summarized on internal wiki: http://crcs.cz/wiki/doku.php?id=research:boinc:boinc-applications-list.

The only working and unused application is Test. But running anything using this application lead to error even in case run, where everything passed.

Also the used computation workflow, that uses our php administration for running experiments is currently unstable. 5th November, Centaur crashed, we restarted it and than it worked somehow OK for 1 day, now is again boinc administration unavailable.

I'm giving this idea of parallel project up. I hope, that Mirek will found alternative way for running up-to-date boinc server on virtual machine for computation in laboratory. And I'm looking forward for computation on metacentrum directly, witout boinc :).
